api/types/network/network.go

```diff
@@ -11,6 +11,7 @@ type IPAM struct {
 	Driver  string
 	Options map[string]string //Per network IPAM driver options
 	Config  []IPAMConfig
+	Data    []IPAMConfig // IPAM operational data
```

Comment:
`Operational`, `OperationalData`, `OperationalState` ?
I am not good at names either. We need to somehow convey this info is the ipam operational state.
Hopefully somebody can propose a good one.